src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/LocationGroup.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.whereabouts.model
+
+import io.swagger.annotations.ApiModel
+import io.swagger.annotations.ApiModelProperty
+import javax.validation.constraints.NotBlank
+import javax.validation.constraints.NotNull
+
+/**
+ * Cell Locations are grouped for unlock lists as a 2 level tree. The two levels are referred to as Location and Sub-Location in the prisonstaffhub UI. Each (location/sub-location) group has a name that is understood by prison officers and also serves as a key to retrieve the corresponding Cell Locations and information about their occupants.
+ */
+@ApiModel(description = "Cell Locations are grouped for unlock lists as a 2 level tree. The two levels are referred to as Location and Sub-Location in the prisonstaffhub UI. Each (location/sub-location) group has a name that is understood by prison officers and also serves as a key to retrieve the corresponding Cell Locations and information about their occupants.")
+data class LocationGroup (
+  @ApiModelProperty(required = true, value = "The name of the group", example = "Block A")
+  val name: @NotBlank String,
```

Comment:
again what is validating these annotations?

Reply:
Nothing - gone.